recordablesurfaceview/src/main/java/com/uncorkedstudios/android/view/recordablesurfaceview/RecordableSurfaceView.java

```diff
@@ -397,6 +406,7 @@ public void setRendererCallbacks(RendererCallbacks surfaceRendererCallbacks) {
 
         EGLSurface mEGLSurfaceMedia;
 
+        Queue<Runnable> mRunnableQueue = new PriorityQueue<>();
```

Comment:
Seems like using a priority queue without allowing the user to specify a comparator may have unexpected behavior.  As a user, I would expect my events to run FIFO, however events that are added to this queue will be run in an order determined by Runnable.compareTo().

Reply:
So I get what you're saying here, so I'm going to cut over to a LinkedList since the head object is easily poppable with the same set of calls. I guess the weird thing here is that it has to be strictly FIFO since there's no facility to prioritize or hint prioritization of Runnable :\